documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -31,13 +31,28 @@ DefaultCacheManager manager = new DefaultCacheManager(gcb.build());
 Individual caches can then be configured in different modes:
 
 * *Local*: changes and reads are never replicated. This is the only mode available in
-standalone cache managers.
+non-clustered cache managers.
 * *Invalidation*: changes are not replicated, instead the key is invalidated on all
 nodes; reads are local.
 * *Replicated*: changes are replicated to all nodes, reads are always local.
 * *Distributed*: changes are replicated to a fixed number of nodes, reads request the
 value from at least one of the owner nodes.
 
+=== Which cache mode should I use?
+
+Which cache you should use depends on the qualities/guarantees you need for your data. The following table summarizes the most important ones:
+
+[cols="20,^15,^15,^15,^15,^15",options="header"]
+|============================================================
+|                  | Simple      | Local       | Invalidation | Replicated | Distributed
+| Clustered        | [red]*N*    | [red]*N*    | [green]*Y*   | [green]*Y* | [green]*Y*
+| Read performance | [green]*High* | [green]*High* | [green]*High* | [green]*High* | [yellow]*Medium*
+| Write performance| [green]*High* | [green]*High* | [green]*High* | [red]*Low* | [yellow]*Medium*
```

Comment:
In invaliadation cache writes aren't are definitely not High since these synchronously contact all nodes (so it's rather Low to Medium)